examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -24,40 +24,30 @@
  **/
 #include "DeviceCallbacks.h"
 
-#include "CHIPDeviceManager.h"
 #include "Globals.h"
 #include "LEDWidget.h"
 #include "WiFiWidget.h"
-#include "esp_bt.h"
-#include "esp_check.h"
-#include "esp_err.h"
-#include "esp_heap_caps.h"
-#include "esp_log.h"
-#include "esp_nimble_hci.h"
-#include "nimble/nimble_port.h"
-#include "route_hook/esp_route_hook.h"
 #include <app-common/zap-generated/attribute-id.h>
 #include <app-common/zap-generated/attribute-type.h>
-#include <app-common/zap-generated/cluster-id.h>
 #include <app/CommandHandler.h>
 #include <app/clusters/identify-server/identify-server.h>
-#include <app/server/Dnssd.h>
 #include <app/util/basic-types.h>
 #include <app/util/util.h>
+#include <common/CHIPDeviceManager.h>
 #include <lib/dnssd/Advertiser.h>
-#include <lib/support/CodeUtils.h>
-#include <ota/OTAHelper.h>
+
+#if CONFIG_DEVICE_TYPE_ESP32_C3_DEVKITM
+#include <app-common/zap-generated/cluster-id.h>
```

Comment:
Previously it was not under a config option. Why now?

Reply:
It is needed for esp32-c3 and not for esp32 so added in config option.